src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -756,21 +819,26 @@ private static int compareImports(String import1, String import2) {
     }
 
     /**
-     * Counts empty lines before given.
-     * @param lineNo
-     *        Line number of current import.
-     * @return count of empty lines before given.
+     * Counts empty lines between given parameters.
+     * @param fromLineNo
+     *        One-based line number of previous import.
```

Comment:
What does `one-based` mean? Normal numbers are 10-based. Are you saying this takes in binary number only?

Reply:
The `one-based` was taken from https://github.com/checkstyle/checkstyle/blob/b2ad65928ec8ee3e76e5a1255a26d0ad85bcffbb/src/main/java/com/puppycrawl/tools/checkstyle/api/LineColumn.java#L30-L31 .

I think it is `one-based indexing` more precise.

Reply:
Ok. Lets leave it for now.